src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/BlazorDebugConfigurationProvider.ts

```diff
@@ -0,0 +1,64 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+
+import { RazorLogger } from './RazorLogger';
+
+export class BlazorDebugConfigurationProvider implements vscode.DebugConfigurationProvider {
+    private logger: RazorLogger;
+    private vscodeType: typeof vscode;
+
+    constructor(logger: RazorLogger, vscodeType: typeof vscode) {
+        this.logger = logger;
+        this.vscodeType = vscodeType;
+    }
+
+    public resolveDebugConfiguration(folder: vscode.WorkspaceFolder | undefined, configuration: vscode.DebugConfiguration): vscode.ProviderResult<vscode.DebugConfiguration> {
+        const app = {
+            name: '.NET Core Launch (Blazor Standalone)',
+            type: 'coreclr',
+            request: 'launch',
+            program: 'dotnet',
+            args: ['run'],
+            cwd: '${workspaceFolder}',
+            env: {
+                ASPNETCORE_ENVIRONMENT: 'Development',
+                ...configuration.env,
+            },
+        };
+        const browser = {
+            name: '.NET Core Debug Blazor Web Assembly in Browser',
+            type: configuration.browser === 'edge' ? 'edge' : 'pwa-chrome',

```

Comment:
If you'd like to stick with js-debug for both browsers, `pwa-chrome` and `pws-msedge` work

Reply:
Updated -- thanks for the rec.